src/tool_operate.c

```diff
@@ -1716,20 +1716,31 @@ static CURLcode single_transfer(struct GlobalConfig *global,
 
           {
             long mask =
-              (config->ssl_allow_beast ? CURLSSLOPT_ALLOW_BEAST : 0) |
+              (config->ssl_allow_beast ?
+               CURLSSLOPT_ALLOW_BEAST : 0) |
+              (config->ssl_no_revoke ?
+               CURLSSLOPT_NO_REVOKE : 0) |
               (config->ssl_revoke_best_effort ?
```

Comment:
Why reorder `no_revoke`?

Reply:
I don't remember but my guess is I put them in the order that they were introduced so it would be easier to be checked in the future. I will add your other suggestions. Thanks